style.css

```
@@ -0,0 +1,174 @@
+@import url(common.css);
+
+body {
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400; 
+  line-height: normal;
+}
+a {
+  text-decoration: none;
+  cursor: pointer;
+}
+.cta {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 16px 20px;
+  border-radius: 8px;
+  background: var(--gra-purpleblue-to-skyblue);
+  color: var(--grey-light);
+  font-size: 18px;
+  font-weight: 600;
+  cursor: pointer;
+}
+
+/* Header */
+header {
+  background: var(--linkbrary-bg);
+  padding-top: 95px;
+}
+.gnb {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  padding: 20px 200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: var(--linkbrary-bg);
+  z-index: 100;
+}
+.login {
+  width: 128px;
+}
+.hero {
+  padding: 70px 360px 0px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.hero p {
+  font-size: 4rem;
+  font-weight: 700;
+  line-height: 125%;
+  text-align: center;
+}
+.hero p .hero-gradient {
+  background: linear-gradient(91deg, #6D6AFE 17.28%, #FF9F9F 74.98%);
+  background-clip: text;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+.hero .cta {
+  width: 350px;
+  margin: 40px 0;
+}
+
+/* Contents */
+.content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 50px 461px;
+  gap: 157px;
+}
+.title-wrap {
+  width: 291px;
```

```suggestion
  min-width: 291px;
```
페이지 사이즈가 줄어들면 세로쓰기 형태로 바뀐다고 말씀 드렸는데요, 방지하는 한 가지 방법은 최소 보장 너비를 주는 겁니다. 이렇게 고쳐보시면 페이지가 작아져도 너비가 291보다 작아지지 않는 걸 확인하실 수 있습니다!